test/visualTest.ts

```diff
@@ -123,56 +125,30 @@ describe("DualKpi", () => {
         });
     });
 
-    describe("Capabilities tests", () => {
-        it("all items having displayName should have displayNameKey property", () => {
-            jasmine.getJSONFixtures().fixturesPath = "base";
-
-            let jsonData = getJSONFixture("capabilities.json");
-
-            let objectsChecker: Function = (obj) => {
-                for (let property in obj) {
-                    let value: any = obj[property];
-
-                    if (value.displayName) {
-                        expect(value.displayNameKey).toBeDefined();
-                    }
-
-                    if (typeof value === "object") {
-                        objectsChecker(value);
-                    }
-                }
-            };
-
-            objectsChecker(jsonData);
-        });
-    });
-
     describe("DOM tests", () => {
-        it("svg element created", () => expect(visualBuilder.mainElement[0]).toBeInDOM());
+        it("svg element created", () => expect(visualBuilder.mainElement).toBeDefined());
 
         it("update", (done) => {
             visualBuilder.updateRenderTimeout(dataView, () => {
-                expect(visualBuilder.chartGroup.children("path.area"))
-                    .toBeInDOM();
-                expect(visualBuilder.chartGroup.children("g.axis").children("g.tick").length)
-                    .toBe(4);
+                expect(visualBuilder.pathAreas.length).toBeGreaterThan(0);
+                expect(visualBuilder.axisTicks.length).toBe(4);
```

Comment:
why 4? Please add comments for such strict values to make it clear for a fast understanding from a first glance.

Reply:
Top and bottom charts are shown, 2 axis ticks (lower and upper values) are shown, so 4 axis.
I'll add comments making it clearer.